apps/easypid/src/app/(app)/notifications/offlinePresentation.tsx

```diff
@@ -0,0 +1,20 @@
+import { FunkeMdocOfflineSharingScreen } from '@easypid/features/share/FunkeMdocOfflineSharingScreen'
+import { useLocalSearchParams } from 'expo-router'
+
+export default function Screen() {
+  const { sessionTranscript, deviceRequest, requestedAttributes } = useLocalSearchParams()
+
+  const sessionTranscriptArray = new Uint8Array(Buffer.from(sessionTranscript as string, 'base64'))
+  const deviceRequestArray = new Uint8Array(Buffer.from(deviceRequest as string, 'base64'))
+  const requestedAttributesArray = Array.isArray(requestedAttributes)
+    ? requestedAttributes
+    : JSON.parse(requestedAttributes as string)
```

Comment:
I think the input will never be array right (with the json parse?)

```suggestion
  const { sessionTranscript, deviceRequest, requestedAttributes } = useLocalSearchParams<{ sessionTranscript: string; deviceRequest: string; requestedAttributes: string }>()

  const sessionTranscriptArray = new Uint8Array(Buffer.from(sessionTranscript, 'base64'))
  const deviceRequestArray = new Uint8Array(Buffer.from(deviceRequest, 'base64'))
  const requestedAttributesArray = JSON.parse(requestedAttributes)
```